src/main/java/org/broadinstitute/hellbender/tools/copynumber/CreateBinningIntervals.java

```diff
@@ -0,0 +1,122 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.io.File;
+import java.util.List;
+
+
+
+@CommandLineProgramProperties(
+        summary = "Split intervals into sub-interval files.",
+        oneLineSummary = "Split intervals into sub-interval files.",
+        programGroup = VariantProgramGroup.class
+)
+@DocumentedFeature
+public class CreateBinningIntervals extends GATKTool {
+    public static final String WIDTH_OF_BINS_SHORT_NAME = "bw";
+    public static final String WIDTH_OF_BINS_LONG_NAME = "binwidths";
+
+    public static final String PADDING_SHORT_NAME = "pad";
+    public static final String PADDING_LONG_NAME = "padding";
+
+    @Argument(
+            doc = "width of the bins",
+            fullName = WIDTH_OF_BINS_LONG_NAME,
+            shortName = WIDTH_OF_BINS_SHORT_NAME,
+            optional = true,
+            minValue = 1
+    )
+    private int widthOfBins = 1;
+
+    @Argument(
+            doc = "width of the padding regions",
```

Comment:
See comment below about expanding documentation to mention that intervals will be transformed first by standard command-line arguments.

Reply:
Thanks for this comment, I didn't realize that IntervalArgumentCollection had this flag. Based on our existing tests, '--interval_padding' has the same effect as '-P'. When both of them are used, then -P will create extra padding on top of '--interval_padding'. 

Based on my discussion with @samuelklee, I decided to keep the -P flag. Users who are not very familiar with the command line arguments of IntervalArgumentCollection will probably only use the -P flag. More advanced users will likely use the IntervalArgumentCollection flags, since those allow more flexibility to change the intervals. 

I mentioned in the javadoc that the -P padding is on top of --interval__padding, and that we encourage using the -P flag only.